### ISSUE TITLE
튜닝요소 ( baseline 동영상 강좌속 내용)

### ISSUE THREAD

**Issue by huefilm** [AIBootcamp13/upstageailab-cv-classification-cv_8#5]
- [ ]  EDA
- [x] validation set (8:2로 검증데이터를 만드는것)
- [x] 스케쥴러 추가 (Learning Rate Scheduler)
- [x] 모델변경(baseline기본: ResNet)
- [x] augmentation(baseline기본: Resize, Normalize)
